### ISSUE TITLE
Ability to set a different automated deletion expiry for boosts and toots

### ISSUE THREAD

**Issue by bohwaz** [mastodon/mastodon#21220]
### Pitch

I would like to be able to set a different expiry for things I boosted, than for things I posted myself.

For example: delete boosts after 2 weeks, delete my own toots after 3 months.

### Motivation

Things you create yourself are more important than content that you just boosted.

**Comment by Ghryphen** [comment]
Agreed, I currently use a bot to undo retweets after a week. Since auto deleting is implemented in Mastodon, it'd be nice to have a function to auto remove boosts after a period of time, but I don't want to delete my own posts so would need it's own expiry setting.
